gsa/src/web/pages/reports/details/thresholdpanel.js

```diff
@@ -0,0 +1,168 @@
+  const severity = filter.get('severity', 0);
+
+  const handleRemoveLogLevel = () => {
+    if (levels.includes('g')) {
+      const newLevels = levels.replace('g', '');
+      const lfilter = filter.copy();
+      lfilter.set('levels', newLevels);
+
+      onFilterChanged(lfilter);
+    }
+  };
+
+  const handleRemoveLowLevel = () => {
+    if (levels.includes('l')) {
+      const newLevels = levels.replace('l', '');
+      const lfilter = filter.copy();
+      lfilter.set('levels', newLevels);
+
+      onFilterChanged(lfilter);
+    }
+  };
+
+  const handleRemoveMediumLevel = () => {
+    if (levels.includes('m')) {
+      const newLevels = levels.replace('m', '');
+      const lfilter = filter.copy();
+      lfilter.set('levels', newLevels);
+
+      onFilterChanged(lfilter);
+    }
+  };
+  const handleSetMinimumSeverity = () => {
+    const lfilter = filter.copy();
+    lfilter.set('severity', 7.0, '>');
+
+    onFilterChanged(lfilter);
+  };
+  return (
+    <UpdatingDivider
+      flex="column"
+      align={['start', 'stretch']}
+      grow
+      isUpdating={isUpdating}
+    >
+      <InfoPanel
+        heading={_(
+          "The {{entityType}} can't be displayed, because the " +
+            'report contains to many results. Please decrease the number ' +
+            'of results by applying a different filter.',
+          {entityType},
+        )}
+      />
+      <Divider wrap>
+        {levels.includes('g') && (
+          <ReportPanel
+            icon={props => <FilterIcon {...props} />}
+            title={_('Log messages are currently included.')}
+            onClick={handleRemoveLogLevel}
+          >
+            {_('Exclude log messages in your filter settings.')}
+          </ReportPanel>
+        )}
+        {levels.includes('l') && (
+          <ReportPanel
+            icon={props => <FilterIcon {...props} />}
+            title={_('Results with low severity are currently included.')}
+            onClick={handleRemoveLowLevel}
+          >
+            {_('Exclude results with low severity in your filter settings.')}
+          </ReportPanel>
+        )}
+        {levels.includes('m') && (
+          <ReportPanel
+            icon={props => <FilterIcon {...props} />}
+            title={_('Results with medium severity are currently included.')}
+            onClick={handleRemoveMediumLevel}
+          >
+            {_('Exclude results with medium severity in your filter settings.')}
+          </ReportPanel>
+        )}
+        {!filter.has('levels') && severity < 7 && (
+          <ReportPanel
+            icon={props => <FilterIcon {...props} />}
+            title={_("Results aren't filtered by severity.")}
+            onClick={handleSetMinimumSeverity}
+          >
+            {_('Apply a minimum severity of 7.0.')}
+          </ReportPanel>
+        )}
+        <ReportPanel
+          icon={props => <EditIcon {...props} />}
+          title={_('Your filter settings may be too unrefined.')}
```

Comment:
This sounds a bit like blaming the user, but this may only be true for the German translation (it might also mean unraffiniert, unqualifiziert). Maybe use something more neutral.

```suggestion
          title={_('Your filter settings may be too coarse.')}
```